docs/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,37 @@
+---
+name: New pull request
+about: Create a new pull request
+title: "feat: <cool_name>"
+labels:
+assignees:
+---
+
+## Description
+<!-- Describe your changes in detail -->
+
+## Motivation and Context
+<!-- Why is this change required? What problem does it solve? -->
+<!-- If it fixes an open issue, please link the issue here. e.g fix #4242 -->
+
+## How Has This Been Tested?
+<!-- Please describe in detail how you tested your changes. -->
+<!-- Include details of your testing environment, and the tests you ran to -->
+<!-- see how your change affects other areas of the code, etc. -->
+
+## Screenshots (if appropriate):
+
+## Types of changes
+<!-- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to change)
+
+## Checklist:
+<!-- Go over all the following points, and put an `x` in all the boxes that apply. -->
+<!-- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
+- [ ] My code follows the code style of this project.
+- [ ] My change requires a change to the documentation.
+- [ ] I have updated the documentation accordingly.
+- [ ] I have read the **[CONTRIBUTING](https://github.com/berty/community/blob/master/CONTRIBUTING.md)** document.
+- [ ] I have added tests to cover my changes.
+- [ ] All new and existing tests passed.
```

Comment:
I think that this is very very long and annoying

If someone is opening a PR (already made some effort to make a contribution), I think we should not make his/her life harder but just start reviewing and give some feedback

And if we find that we regularly ask the same question or give the same feedback, then we can setup a bot that will automatically check for the common errors and add a comment on the fly